datumaro/components/format_detection.py

```diff
@@ -35,14 +35,21 @@ class FormatDetectionConfidence(IntEnum):
     """
 
     NONE = 1
+    """
+    """
+    EXTREME_LOW = 5
+    """
+    EXTREME_LOW: This is currently only assigned to ImageDir format. This is because
+    ImageDir format can be detected in every image dataset format.
+    """
     LOW = 10
     """
-    The dataset seems to belong to the format, but the format is too loosely
+    LOW: The dataset seems to belong to the format, but the format is too loosely
     defined to be able to distinguish it from other formats.
     """
     MEDIUM = 20
     """
-    The dataset seems to belong to the format, and is likely not to belong
+    MEDIUM: The dataset seems to belong to the format, and is likely not to belong
```

Comment:
Do you have any plan to introduce `HIGH`?
Could you let me know what number (e.g., 1, 5, 10, and 20) means here?

Reply:
There is no plan for high, and extreme low was introduced to minimize change while maintaining momentum. The absolute number do not currently mean much, only relative differences between them exist, so that those. Therefore, the absolute numbers are given arbitrarily.